etl-job/fhir_import_export.py

```diff
@@ -327,9 +342,6 @@ def _empty_project(output: dict,
             meta_flat_delete(project_id=f"{program}-{project}", index=index)
         output['logs'].append(f"EMPTIED flat for {program}-{project}")
 
-        fhir_delete(f"{program}-{project}", DEFAULT_ELASTIC)
```

Comment:
Question:  if a project is deleted, how is it emptied from the grip graph?

Reply:
It is part of the empty_project method:
https://github.com/ACED-IDP/aced_etl_pod/blob/d2c455c8ddfdc3f340c8268ddf86c8792088a758/etl-job/fhir_import_export.py#L337-L339 calls 
https://github.com/ACED-IDP/submission/blob/development/aced_submission/grip_load.py#L212-L221 which then calls grip server plugin method:
https://github.com/bmeg/grip-graphql/blob/main/gen3_writer/handler.go#L490-L521

Reply:
thanks